plugins/browser-plugin-media-tracking/src/buildMediaEvent.ts

```diff
@@ -60,7 +60,7 @@ function getHTMLMediaElementEntities(el: HTMLVideoElement | HTMLAudioElement): M
       readyState: READY_STATE[el[MediaProperty.READYSTATE]] as MediaElement['readyState'],
       seekable: timeRangesToObjectArray(el[MediaProperty.SEEKABLE]),
       seeking: el[MediaProperty.SEEKING],
-      src: dataUriHandler(el[MediaProperty.SRC]),
+      src: dataUrlHandler(el[MediaProperty.SRC]),
```

Comment:
In the schema (https://github.com/snowplow/iglu-central/blob/236131df62a21ccfa7b21c1a1fa3a8cabc74e39e/schemas/org.whatwg/media_element/jsonschema/1-0-0) `src` is required. However it can now end up being undefined/null if it's a `data:` url. Might need to change the schema as I don't think theres an alternative.